### DISCUSSION TITLE
Browser extending name at multiple downloads breaks exe config

### DISCUSSION THREAD

**Discussion by sysup-at** [rustdesk/rustdesk#7149] [General]
### Bug Description

When a customer downloads RustDesk for quick support multiple times, browser adds (1).exe, (2).exe, ...
This keeps rustdesk from connecting to correct server. Since customers are often not able to execute the more complex steps to resolve this, it is required for rustdesk client to compensate browsers behaviour.

### How to Reproduce

Download a named rustdesk-licensed...exe multiple times using a common browser and execute the second or third download.

### Expected Behavior

RustDesk splits end of name by '(' and uses [0] as config string so browsers name extension gets cut out.

### Operating system(s) on local side and remote side

Windows

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

Not required, problem is clear.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/4215

**Comment by sysup-at** [comment]
So is this diff in 1.2.3 or not? If yes, well it just happened. After customer deleting all downloads and redownload it worked.
